qiita_pet/handlers/api_proxy/tests/test_prep_template.py

```diff
@@ -187,7 +189,11 @@ def test_prep_template_post_req(self):
                           'LinkerPrimerSequence, BarcodeSequence',
                'file': 'update.txt',
                'id': new_id}
-        self.assertEqual(obs, exp)
+        self.assertEqual(obs['status'], exp['status'])
+        self.assertItemsEqual(obs['message'].split('\n'),
```

Comment:
If you pay attention to the code that generates this string, it is a result of joining the strings stored in a set. Since it is in a set, it is unordered, so we cannot ensure the order of the strings here. By doing the split and assertItemsEqual I can compare that the contents are the same.